### PR TITLE
Corrected Linguist README

### PR DESCRIPTION
**Pull request by ahhhndre**
## Hey, I just made a Pull Request!

In PR #14718 that added the Linguist plugin we decided to remove the refresh feature to be added back at a later time. Unfortunately the frontend README was not updated. This PR corrects this. 

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by ahhhndre** [comment]
Hi @Rugvip, any chance you can give this a quick approval and merge? Just some documentation updates to reflect the current state, don't want people looking for a feature that isn't in place yet. Thanks in advance! 🚀

File: .changeset/weak-coats-allow.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-linguist': patch
+---
+
+Updated `README.md` to removed images with refresh button as this feature was not implemented in the current version
```
Comment (adamdmharvey) [inline .changeset/weak-coats-allow.md:5]: If it's a READMe only update, you likely don't need a changeset. That triggers a release, which isn't necessary in this case?

**Review by adamdmharvey — COMMENTED** [review]
Think you can remove changeset; otherwise the change looks good given your description!

File: .changeset/weak-coats-allow.md (hunk 1 above)
Comment (ahhhndre) [reply .changeset/weak-coats-allow.md:5]: My understanding to update this on NPM is was needed as well

Comment (Rugvip) [reply .changeset/weak-coats-allow.md:5]: Yep! A changeset is needed for README updates to be reflected on NPM

**Review by Rugvip — APPROVED** [review]
Oh interesting, thank you! 😁

**Pull request merged**